src/Parser/BrewmasterMonk/Modules/Features/EnergyWaste.js

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import Module from 'Parser/Core/Module';
+import Combatants from 'Parser/Core/Modules/Combatants';
+import Icon from 'common/Icon';
+import { formatNumber, formatPercentage } from 'common/format';
+import SPELLS from 'common/SPELLS';
+import SpellLink from 'common/SpellLink';
+import StatisticBox, { STATISTIC_ORDER } from 'Main/StatisticBox';
+import Energy, { MAX_ENERGY } from '../Features/Energy';
+
+class EnergyWaste extends Module {
+  static dependencies = {
+    energy: Energy,
+    combatants: Combatants,
+  };
+
+  suggestions(when) {
+    const energyWastedFromBobPerc = (this.energy.energyWastedFromBOB / this.energy.bobCasts) / MAX_ENERGY;
+    
+    this.energy.bobCasts > 0 && 
+    when(energyWastedFromBobPerc).isGreaterThan(0.4)
```

Comment:
Shouldn't this be 0 by itself when bobCasts is 0? (I'd like to avoid optional suggestions outside of due to not having a talent/item/other static setup)